docs/src/inverse_problems/examples/ode_fitting_oscillation.md

```diff
@@ -1,24 +1,16 @@
-# [Parameter Estimation](@id parameter_estimation)
-The parameters of a model, generated by Catalyst, can be estimated using various
-packages available in the Julia ecosystem. Refer
-[here](https://docs.sciml.ai/Overview/stable/highlevels/inverse_problems/) for
-more extensive information. Below follows a quick tutorial of how
-[Optimization.jl](https://docs.sciml.ai/Optimization/stable/) can be used to fit
-a parameter set to data.
+# [Fitting Parameters for an Oscillatory System](@id parameter_estimation)
+In this examples we will use [Optimization.jl](https://github.com/SciML/Optimization.jl) to fit the parameters of an oscillatory system (the Brusselator) to data. Here, special consideration is taken to avoid reaching a local minimum. Instead of fitting the entire time series directly, we will start with fitting parameter values for the first period, and then use those as an initial guess for fitting the next (and then these to find the next one, and so on). Using this procedure is advantageous for oscillatory systems, and enables us to reach the global optimum.
 
 First, we fetch the required packages.
 ```@example pe1
 using Catalyst
-using DifferentialEquations
-using SciMLSensitivity
+using OrdinaryDiffEq
 using Optimization
-
-# for the ADAM optimizer
-using OptimizationOptimisers
+using OptimizationOptimisers # Required for the ADAM optimizer.
+using SciMLSensitivity # Required for `Optimization.AutoZygote()` automatic differentiation option.
```

Comment:
```suggestion
using OptimizationOptimisers # for the ADAM optimizer
using SciMLSensitivity # for Optimization.AutoZygote() AD backend
```

Reply:
I prefer the old version. Especially spelling out the AD part (As I imagine most people are unfamiliar with this, and especially just "AD backend" doesn't say much.

Reply:
Also, I generally try to make my comments full sentences, not sure if there is some standard regarding this though.